app/components/pages/ThankYou/ThankYou.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import PropTypes from 'prop-types'
+import { H1 } from '@pubsweet/ui'
+import { th } from '@pubsweet/ui-toolkit'
+import ExternalLink from '../../ui/atoms/ExternalLink'
+import ButtonLink from '../../ui/atoms/ButtonLink'
+import Paragraph from '../../ui/atoms/Paragraph'
+import SmallParagraph from '../../ui/atoms/SmallParagraph'
+
+const CenteredContent = styled(Box)`
+  text-align: center;
+`
+const SubText = styled(SmallParagraph)`
+  color: ${th('colorTextSecondary')};
+`
+
+const BookmarkLink = styled(ExternalLink)`
+  display: inline-block;
+`
+
+const ThankYou = ({ title }) => (
+  <CenteredContent mx="auto" width={[1, 1, 1, 600]}>
+    <H1>Thank you</H1>
+    <Paragraph>{`Your submission, "${title}" has been received.`}</Paragraph>
+    <Paragraph>You will be informed of a decision soon.</Paragraph>
+    <SubText>
+      You may want to bookmark the link below to check the progress of your
+      submission.
+      <BookmarkLink href="/">elifesciences.org/submissions</BookmarkLink>
+    </SubText>
+    <ButtonLink primary to="/">
+      Finish
+    </ButtonLink>
+  </CenteredContent>
+)
+
+ThankYou.propTypes = {
+  title: PropTypes.string,
```

Comment:
Might it be more accurate to make this required, rather than adding "Untitled manuscript" as a default?

If I start submitting a manuscript & go to http://localhost:3000/thankyou/3b9b39a0-08c9-430d-8c4c-cd94aa722f38, then I see this:
![image](https://user-images.githubusercontent.com/15656538/46956377-1a98fc80-d08d-11e8-9ff6-ff7b431d9c99.png)

(i.e. an empty string, rather than saying 'untitled...')

'Untitled manuscript' makes sense on the dashboard, if they've only partially completed a submission. But a user shouldn't be able to get to this page without having entered a title, right?

Reply:
This should be the very last page they see after completing the wizards steps, and as the title fields mandatory you're correct they should never get to this page without a title. I'll make it required.